src/forms/brownbag-submissions.njk

```diff
@@ -56,42 +56,300 @@ permalink: brownbag-idea/
           <small id="emailHelp" class="form-text text-muted">Required. We'll never share your email with anyone else.</small>
         </div>
       </fieldset>
+
       <fieldset>
-        <legend>Your Brownbag Idea:</legend>
+        <legend>Your Brownbag Idea</legend>
+        <p class="text-muted">This section is to gather some general information about the talk you're proposing. Don't worry if you don't have all the details ironed out yet—we can work with you to finalize your talk's title and description!</p>
         <div class="form-group">
-          <label for="topicTitle">Title of the Brownbag</label>
+          <label for="topicTitle">What is the proposed title of your talk?</label>
           <input
             type="text"
             class="form-control"
             id="topicTitle"
             name="topicTitle"
-            required
           />
         </div>
         <div class="form-group">
-          <label for="topicIdea">Description we can share on the event page.</label>
-          <textarea class="form-control" required id="topicIdea" name="topicIdea" rows="3"></textarea>
+          <label for="topicDescription">What is the description of your talk?</label>
```

Comment:
I see the name/id changed in this commit, will this be a blocker @danieltott? Also, looks like the 'required' was dropped. Are we not requiring a description? I think something should be required for the description.. but looking at other form fields it doesn't look like we are requiring anything. Should we?

Reply:
My thought on this is that many people who are submitting have ideas about what they'll talk about but haven't finalized them. So requiring _some_ fields would definitely be a preventative measure. For example, requiring a bio or headshot at this stage does cut back on some of the data gathering, but for members who don't have those things available, that means they don't turn in the form. Even asking the format could be premature here IMO.

For the podcast, I send out the email and calendar invite at the same time. In the email, I let them know what they need, and what I'll need by that time. If I don't have that three days before the event, I slack them. 

So my suggestion here would be to add language to say, before the brownbag, please send us x,y, and z. Or to tack on to that, _or_ if you have them available now, here are the fields.